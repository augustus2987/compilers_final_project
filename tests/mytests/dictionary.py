x = {5: 15, 6: True, 10: [3,2,1]}
print x
