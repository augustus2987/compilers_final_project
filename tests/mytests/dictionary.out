{10: [3, 2, 1], 5: 15, 6: True}
